src/features/list/dash_js.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MediaSourceContentInitializer from "../../main_thread/init/media_source_content_initializer";
+import mainCodecSupportProber from "../../mse/main_codec_support_prober";
+import dashFastJsParser from "../../parsers/manifest/dash/fast-js-parser";
+import dash from "../../transports/dash";
+import type { IFeaturesObject } from "../types";
+
+/**
+ * Add ability to play DASH contents.
+ * @param {Object} features
+ */
+function addDASHFeature(features: IFeaturesObject): void {
+  if (features.transports.dash === undefined) {
+    features.transports.dash = dash;
+  }
+  features.dashParsers.fastJs = dashFastJsParser;
+  features.mainThreadMediaSourceInit = MediaSourceContentInitializer;
+  features.codecSupportProber = mainCodecSupportProber;
+}
+
+export { addDASHFeature as DASH };
```

Comment:
The feature as same name than `src/features/list/dash`
Therefore it's not exported through the index at `src/features/list/index`, so I imagine an API user cannot add it to it's feature with:

```js
import { DASH_JS } from "rx-player/features"
RxPlayer.addFeatures([DASH_JS])
```
What the idea behind this ? Maybe we have in mind that the FastJS parser could replace the DOM parser and therefore we don't want to expose it publicly as in a future version this will be the default and we don't want to add some overhead to users? 

I'm wondering if this feature should be flagged as experimental while we are determining if this should replace DOM parser or not.

Reply:
Yeah I don't know where I was going with this :D  

For now, it was only intended to be worker-only, as an application wouldn't be sure if it wants the native or fast-js flavor (and would have relied on the WebAssembly parser if perf was an issue).

So we may just want to remove that file for now.

Reply:
Removed

Reply:
Ok understood